content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -15,251 +15,102 @@ owner: istio/wg-environments-maintainers
-   /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
-   the scope of this document.
+### Send requests from virtual machine workloads to Kubernetes services
 
-1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
+You can send traffic to `httpbin.default.svc.cluster.local` and get a response from the server. You must configure DNS in `/etc/hosts` to map the `httpbin.default.svc.cluster.local` domain name to an IP, or the IP will not resolve. In this case, the IP should be an IP that is routed over the single network using L3 connectivity. You should use the IP of the service in the Kubernetes cluster.
 
     {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+    $ curl -v httpbin.default.svc.cluster.local:8000/headers
     {{< /text >}}
 
-1.  Install `root-cert.pem` under `/var/run/secrets/istio/`.
+### Running services on the virtual machine
 
-1.  Install `cluster.env` under `/var/lib/istio/envoy/`.
+1. Setup an HTTP server on the virtual machine to serve HTTP traffic on port 8080:
 
     {{< text bash >}}
-    $ sudo cp cluster.env /var/lib/istio/envoy
-    {{< /text >}}
-
-1.  Transfer ownership of the files in `/etc/certs/` , `/var/lib/istio/envoy/` and `/var/run/secrets/istio/`to the Istio proxy.
-
-    {{< text bash >}}
-    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy /var/run/secrets/istio/
-    {{< /text >}}
-
-1.  Start Istio using `systemctl`.
-
-    {{< text bash >}}
-    $ sudo systemctl start istio
-    {{< /text >}}
-
-## Send requests from VM workloads to Kubernetes services
-
-After setup, the machine can access services running in the Kubernetes cluster
-or on other VMs.
-
-The following example shows accessing a service running in the Kubernetes cluster from a VM using
-`/etc/hosts/`, in this case using a service from the [Bookinfo example](/docs/examples/bookinfo/).
-
-1.  First, on the cluster admin machine get the virtual IP address (`clusterIP`) for the service:
-
-    {{< text bash >}}
-    $ kubectl get svc productpage -o jsonpath='{.spec.clusterIP}'
-    10.55.246.247
-    {{< /text >}}
-
-1.  Then on the added VM, add the service name and address to its `etc/hosts`
-    file. You can then connect to the cluster service from the VM, as in the
-    example below:
-
-    {{< text bash >}}
-$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
-$ curl -v productpage.default.svc.cluster.local:9080
-< HTTP/1.1 200 OK
-< content-type: text/html; charset=utf-8
-< content-length: 1836
-< server: envoy
-... html content ...
+    $ python -m SimpleHTTPServer 8080
     {{< /text >}}
 
-The `server: envoy` header indicates that the sidecar intercepted the traffic.
+    {{< idea >}}
+    Note, you may have to open firewalls to be able to access the 8080 port on your virtual machine
+    {{< /idea >}}
```

Comment:
```suggestion
    {{< warning >}}
    Note, you may have to open firewalls to be able to access the 8080 port on your virtual machine
    {{< /warning >}}
```

Reply:
I removed the word note, but otherwise applied your comment as suggested.